src/components/VideoPlayerPreview/VideoPlayerThumbnail.tsx

```diff
@@ -12,20 +11,9 @@ import * as DeviceCapabilities from '@libs/DeviceCapabilities';
 import * as ReportUtils from '@libs/ReportUtils';
 import variables from '@styles/variables';
 import CONST from '@src/CONST';
+import type {VideoPlayerThumbnailProps} from './types';
 
-const propTypes = {
-    onPress: PropTypes.func.isRequired,
-
-    accessibilityLabel: PropTypes.string.isRequired,
-
-    thumbnailUrl: PropTypes.string,
-};
-
-const defaultProps = {
-    thumbnailUrl: undefined,
-};
-
-function VideoPlayerThumbnail({thumbnailUrl, onPress, accessibilityLabel}) {
+function VideoPlayerThumbnail({thumbnailUrl = undefined, onPress, accessibilityLabel}: VideoPlayerThumbnailProps) {
```

Comment:
No need to use undefined default value as it serves no purpose (it's undefined either way)